src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -202,7 +192,7 @@ def get(self, request: Request, project: Project) -> Response:
             # Post-processing before exiting scope context
             found: bool = result["sourceUrl"] is not None
             scope.set_tag("stacktrace_link.found", found)
-
+            scope.set_tag("stacktrace_link.auto_derived", derived)
```

Comment:
Shortening from `stacktrace_link.automatically_generated` since there's a limit of 32 chars.